modules/navier_stokes/src/utils/FaceCenteredMapFunctor.C

```diff
@@ -160,5 +160,7 @@ FaceCenteredMapFunctor<T, Map>::evaluate(const FaceInfo * fi) const
 
 template class FaceCenteredMapFunctor<ADRealVectorValue,
                                       std::unordered_map<dof_id_type, ADRealVectorValue>>;
+template class FaceCenteredMapFunctor<ADRealVectorValue,
+                                      std::unordered_map<dof_id_type, RealVectorValue>>;
```

Comment:
the AD and nonAD functor interface dont do the conversion for you?

Reply:
Ahh yes, they do, when this PR started we didn't have it. Nice catch!